db-models/comments.sql

```diff
@@ -0,0 +1,9 @@
+-- db-models/comments.sql
+
+CREATE TABLE comments (
+    comment_id SERIAL PRIMARY KEY,          -- Unique for each comment
+    parent_comment_id INTEGER,              -- ID of the parent comment for replies
+    author_name TEXT NOT NULL,              -- Author comment's name
```

Comment:
I think a better way to include the author name is to actually use something like `creator_id INTEGER` which will be a "foreign key" to the eventual `users` table. That way, we can do a query like `SELECT * FROM COMMENTS JOIN WHERE creator_id = user_id` which will join the data together and we can get the author name directly from the user row.

SQLite foreign keys: https://www.geeksforgeeks.org/sqlite-foreign-key/